README.md

```diff
@@ -14,21 +15,61 @@ cd parquet-site
 git submodule update --init --recursive
 ```
 
-To build or update your site’s CSS resources, you also need PostCSS to create the final assets. By default npm installs tools under the directory where you run npm install.
+To build or update CSS resources, you also need PostCSS to create the final assets.  By default npm installs tools under the directory where you run npm install.
 
 ```
 npm install -D autoprefixer
 npm install -D postcss-cli
 npm install -D postcss
 ```
 
-To run this website site locally, run the following in the root of the directory:
+To preview this website site locally, run the following in the root of the directory:
 
 ```shell
 hugo server
 ```
 
-# Release Documentation
+## Building and Running in Docker
+
+If you don't want to install `hugo` and its dependencies local machine, you can use
+docker to preview locally.  First checkout the `parquet-site` explained above 
+and then run:
+
+```shell
+# run docker container mounting the current directory to /parquet-site and exposing port 1313
+docker run -it -v `pwd`:/parquet-site -p 1313:1313  debian:bullseye-slim  bash
```

Comment:
Is it better to use a dockerfile which is much easier to use? I'm just asking but not required to change. These steps are helpful enough.

Reply:
A docker file is a good idea. I will make one